app/assets/stylesheets/custom.scss

```diff
@@ -91,3 +101,52 @@ footer {
     }
   }
 }
+
+.debug-dump {
+  @include box-size;
+  clear: both;
+  float: left;
+  margin-top: 45px;
+  width: 100%;
+}
+
+// forms
+
+select {
```

Comment:
k nên viết thế này. sử dụng class đi. vì nếu dùng thế này toàn app với tất cả select sẽ bị ảnh hưởng. rất nguy hiểm